go.mod

```diff
@@ -204,7 +204,7 @@ require (
 	github.com/stretchr/objx v0.3.0 // indirect
 	github.com/subosito/gotenv v1.2.0 // indirect
 	github.com/sylvia7788/contextcheck v1.0.4 // indirect
-	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca // indirect
+	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca
```

Comment:
I'm guessing this wasn't updated when the force prune PR was merged

Reply:
It should be moved into the first require stanza: https://github.com/osmosis-labs/osmosis/blob/e9d877b1f623fc36344b6fc3a9635d3ce3ebe186/go.mod#L5-L30

Reply:
Basically, the force prune PR made our codebase directly depend on leveldb. I think that's why the indirection comment was removed. However, it wasn't auto moved into the first set of requires that we use for direct dependencies

Reply:
Ahh I was wondering why it kept removing the indirect comment when there were no changes to this, thanks for explaining